src/data/icons/svg/XDATA.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="64px" height="64px" viewBox="0 0 64 64" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 49.3 (51167) - http://www.bohemiancoding.com/sketch -->
+    <title>64px</title>
+    <desc>Created with Sketch.</desc>
+    <defs></defs>
+    <g id="64px" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
```

Comment:
Thanks for the contribution. could you merge master in? we have extra test that do some checks to ensure icons are fine.

Unfortunately, we can't afford to have `id=`, `class=` or `<style>` in these SVG because we will convert them back into react class.